### ISSUE TITLE
Add DEB, RPM and maybe other packages

### ISSUE THREAD

**Issue by krom** [TheTinkerDad/sensible#15]
After adding systemd support I propose to add building DEB and RPM packages for Linux distributions.
I plan do fork and prepare PR with configuration for building packages.

But I see what you have no automatic build configuration. By using github actions/workflow you can prepare release from tag.

**Comment by TheTinkerDad** [comment]
Yes, this is something I'm planning to do. Thanks for the PRs in advance!
